aten/src/ATen/native/xnnpack/GlobalAveragePooling.cpp

```diff
@@ -0,0 +1,75 @@
+#ifdef USE_XNNPACK
```

Comment:
Can you name the file AveragePooling with GlobalAveragePool just a special case?